client/src/utils/helpers.js

```diff
@@ -161,6 +161,17 @@ export const encounterLetterToTypeConversion = (encounter) => {
   }
 };
 
+export const paymentMethodType = (type) => {
+  switch (type) {
+    case "V":
+      return "Visa";
+    case "M":
+      return "Master";
+    default:
+      return "";
+  }
```

Comment:
Sure, let me do it this way.

Reply:
I think it's better to stay it like this because if the backend returns an empty string on any other value than "V" or "M" then our default case will return a value let's say Visa or Master which is a wrong intuition for the user.